tools/chapel-py/src/core-types-gen.h

```diff
@@ -0,0 +1,92 @@
+/*
+ * Copyright 2023-2024 Hewlett Packard Enterprise Development LP
```

Comment:
Just noting now that this new file should be 2024-2024

Reply:
I don't think so. I copied code that was written under 2023-2024 copyright -- which I think makes it still have the same copyright here.